docs/object.md

```diff
@@ -1480,7 +1487,28 @@ newDecks[4].deal(1)
 
 
 ---
+####spread(...)
 
+[<span class="ret tab"></span>](types.md)&nbsp;Spreads the cards of a deck out on the table.
+
+!!!important
+	Cards take a frame to be created. This means trying to act on them immediately will not work. Use a coroutine or timer to add a delay.
+
+!!!info "spread(distance)"
+	* [<span class="tag flo"></span>](types.md) **distance**: How far apart should the cards be.
+		* {>>Optional, if no value is provided, they will be 0.6 inches apart.<<}
+		* {>> Negative values will spread to the left instead of the right.<<}
+
+!!!info "Returned table"
+	The number of Objects in the table is equal to the number of cards in the deck. They are returned in the order they were in the deck.
+
+	* [<span class="tag tab"></span>](types.md) The table that is returned
+		* [<span class="tag obj"></span>](types.md) **1**: The first card in the deck
+		* [<span class="tag obj"></span>](types.md) **2**: The second card in the deck
+		* [<span class="tag obj"></span>](types.md) **3**: The third card in the deck (etc)
```

Comment:
deck, etc.

Reply:
I modeled it after split, should that be changed too?